src/protocols/routing/mod.rs

```diff
@@ -16,6 +16,8 @@ pub use self::forward::ParsedForward;
 
 pub(crate) const FORWARD_MSG_TYPE: &str = "https://didcomm.org/routing/2.0/forward";
 
+pub(crate) const DIDCOMM_V2_PROFILE: &str = "didcomm/v2";
```

Comment:
We also need to check that Forward protocol version is correct (2.x), see https://github.com/sicpa-dlab/didcomm-python/blob/main/didcomm/core/validators.py#L41.
Let's do it later in the scope of https://github.com/sicpa-dlab/didcomm-gemini/issues/111